business_requirement_deliverable_project_categ/README.rst

```diff
@@ -0,0 +1,209 @@
+    |
+    |- DL2
+    |  |
+    |  |- RL3
+    |  `- RL4
+    |
+    |- DL3
+    |  |
+    |  |- RL5
+    |  `- RL6
+    |
+    |- BR3 (Sub BR of BR2)
+       |
+       `- DL4
+          `- RL7
+    
+.. figure:: static/img/bus_req_project.png
+   :width: 600 px
+   :alt: Business Requirement Project Wizard
+
+In the Project creation wizard (above image), if no option is selected:
+
+* No sub project created
+* 7 tasks are created in the master project (for RL1~RL7)
+
+If option 1 is selected:
+
+* 3 sub projects (parent is Master project), one per BR
+* 7 tasks are created for
+
+  * in project linked to BR1 (2 tasks for RL1 and RL2),
+  * in project linked to BR2 (4 tasks for RL3~6),
+  * in project linked to BR3 (1 task for RL7)
+
+If option 2 is selected:
+
+* 4 sub projects (parent is Master project), one per DL
+* 7 tasks are created for
+
+  * in project linked to DL1 (2 tasks for RL1 and RL2),
+  * in project linked to DL2 (2 tasks for RL3~4),
+  * in project linked to DL3 (2 tasks for RL5~6) and
+  * in project linked to DL4 (1 task for RL7)
+
+If options 1+2 are selected:
+
+* 7 sub projects (parent is Master project), one per BR and one per DL
+
+  * BR1 sub-project contains DL1 sub-project,
+  * BR2 sub-project contains DL2 and DL3 sub-projects,
+  * BR3 sub-project contains DL4 sub-project
+
+* 7 tasks are created for
+
+  * in project linked to DL1 (2 tasks for RL1 and RL2),
+  * in project linked to DL2 (2 tasks for RL3~4),
+  * in project linked to DL3 (2 tasks for RL5~6) and
+  * in project linked to DL4 (1 task for RL7)
+
+If options 1+3 are selected:
+
+* 7 sub projects (parent is Master project), one per BR and sub-BR
+
+  * BR1 sub-project
+  * BR2 sub-project contains BR3 sub-projects,
+
+* 7 tasks are created for
+
+  * in project linked to BR1 (2 tasks for RL1 and RL2),
+  * in project linked to BR2 (4 tasks for RL3~6),
+  * in project linked to BR3 (1 task for RL7)
+
+If options 1+2+3 are selected:
+
+* 7 sub projects (parent is Master project), one per BR, sub-BR and DL
+
+  * BR1 sub-project contains DL1 sub-project,
+  * BR2 sub-project contains BR3, DL2 and DL3 sub-projects,
+  * BR3 sub-project contains DL4 sub-project
+
+* 7 tasks are created for
+
+  * in project linked to DL1 (2 tasks for RL1 and RL2),
+  * in project linked to DL2 (2 tasks for RL3~4),
+  * in project linked to DL3 (2 tasks for RL5~6) and
+  * in project linked to DL4 (1 task for RL7)
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/222/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/business-requirement/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eric Caudal<eric.caudal@elico-corp.com>
```

Comment:
Me neither, if Travis could check format of rst files never face before. But indeed I need to fix it

@gurneyalex Could you help us with this question?